bucket/dokan.json

```
@@ -0,0 +1,37 @@
+{
+    "homepage": "https://dokan-dev.github.io/",
+    "description": "Dokan is a user mode file system for Windows. It allows anyone to safely and easily develop new file systems on Windows operating systems.",
+    "license": "LGPL-3.0-only",
+    "version": "1.3.1.1000",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/dokan-dev/dokany/releases/download/v1.3.1.1000/Dokan_x64.msi",
+            "hash": "1b89e2f3cf90902726a5d03a9694596a3a247d6908b74ab493469b259d8f8462"
+        },
+        "32bit": {
+            "url": "https://github.com/dokan-dev/dokany/releases/download/v1.3.1.1000/Dokan_x86.msi",
+            "hash": "b296e7bef4c6ca45f10b6c1b444ff2e38dec40c79753ca24f26f19c1a6aaaf0b"
+        }
+    },
+    "bin": [
+        "dokanctl.exe",
+        "mirror.exe"
+    ],
```

There is no mirror executable.

```suggestion
    "extract_dir": "Dokan\\DokanLibrary-1.3.1",
    "bin": "dokanctl.exe",
```